### PR TITLE
Show shortcuts dialog when pressing `?`

### PR DESCRIPTION
**Pull request by lipis**
### [Preview](https://excalidraw-git-shortcuts-hint.excalidraw.now.sh)

- [x] Use labels from `en.json`
- [x] More shortcuts? Am I missing anything?
- [x] Test on Mac
- [x] Test on Windows
- [x] Test on Linux 
- [x] Footer with links to blog, howto, github
- [x] Scroll if the window height is too small
- [ ] ~~Add a button next to the language select with `?`~~ Next PR

### CODE REVIEW

**Comment by dwelle** [comment]
Works on Windows!

Here's an idea: how about we add a `?` button next to the language picker. And how about we not only list hotkeys, but links to our docs/blog/issue-tracker etc.?

**Comment by lipis** [comment]
How do I update the snapshots?

**Comment by dwelle** [comment]
@lipis 

1. `yarn test`
2. run all (`a`)
3. after failures, type `u`

**Comment by Fausto95** [comment]
`yarn test -u`

File: src/locales/en.json [hunk 1]
```diff
@@ -127,5 +127,14 @@
...
     "title": "Error"
+  },
+  "shortcutsDialog": {
+    "title": "Keyboard shortcuts",
+    "shapes": "Shapes",
+    "editor": "Editor",
+    "view": "View",
+    "blog": "Read the Blog",
+    "howto": "Check out the How To",
+    "github": "Found an issue? Submit"
```
Comment (lipis) [inline src/locales/en.json:138]: I'm out of ideas on what to write here.. what should be the links? Anyone?!

**Comment by dwelle** [comment]
Cool. This is how it looks on Windows:

image

File: src/locales/en.json (hunk 1 above)
Comment (dwelle) [reply src/locales/en.json:138]: blog & github is cool. For the `howto`, maybe:

- "Check out the help guides"
- "Check out our help"

**Comment by dwelle** [comment]
Also, I'd maybe set `target="_blank"` for the links.

File: src/locales/en.json (hunk 1 above)
Comment (kbariotis) [reply src/locales/en.json:138]: - `Read our blog`
- `Follow the guides`
- `See the source code`

Comment (dwelle) [reply src/locales/en.json:138]: @kbariotis the github thing is for bug/issues, not really about source code.

Comment (lipis) [reply src/locales/en.json:138]: Check them out.. I had a combination of the suggestions.

**Comment by dwelle** [comment]
Just realized: the send to back/front on Win should be `Ctrl+Shift+[/]`. Check the source code for the z-action.

**Comment by lipis** [comment]
That's why I wanted them to be the same :/

**Comment by lipis** [comment]
Added the check..!

**Pull request merged**